src/webpack.config.ts

```diff
@@ -23,19 +20,89 @@ module.exports = ({ config }: any) => {
   );
 
   config.module.rules.push({
-    test: /\.(ts|tsx)$/,
+    test: /\.tsx?$/,
     include: [path.resolve(cwd, 'src')],
     use: [
       {
         loader: require.resolve('ts-loader'),
         options: {
+          transpileOnly: true,
           onlyCompileBundledFiles: true, // https://github.com/TypeStrong/ts-loader#onlycompilebundledfiles-boolean-defaultfalse
         },
       },
     ],
   });
 
-  config.resolve.extensions.push('.ts', '.tsx');
+  config.module.rules.push({
+    test: /\.css$/,
+    include: [path.resolve(cwd, 'src')],
+    use: [
+      {
+        loader: require.resolve('style-loader'),
+        options: {
+          sourceMap: true,
+        },
+      },
+      {
+        loader: require.resolve('css-loader'),
+        options: {
+          sourceMap: true,
+          importLoaders: 1,
+        },
+      },
+      {
+        loader: require.resolve('postcss-loader'),
+        options: {
+          sourceMap: true,
+          plugins: [
+            require('autoprefixer')({
+              env: 'last 2 Chrome versions, last 2 Firefox versions, last 1 Safari version',
+            }),
+          ],
+        },
+      },
+    ],
+  });
+
+  config.module.rules.push({
+    test: /\.scss$/,
+    include: [path.resolve(cwd, 'src')],
+    use: [
+      {
+        loader: require.resolve('style-loader'),
+        options: {
+          sourceMap: true,
+        },
+      },
+      {
+        loader: require.resolve('css-loader'),
+        options: {
+          sourceMap: true,
+          importLoaders: 2,
+        },
+      },
+      {
+        loader: require.resolve('postcss-loader'),
+        options: {
+          sourceMap: true,
+          plugins: [
+            require('autoprefixer')({
+              env: 'last 2 Chrome versions, last 2 Firefox versions, last 1 Safari version',
+            }),
+          ],
+        },
+      },
+      {
+        loader: require.resolve('sass-loader'),
+        options: {
+          sourceMap: true,
+          includePaths: [path.resolve(cwd, 'node_modules')],
```

Comment:
You need to include `node-sass-package-importer` as the importer option I believe

Reply:
good catch